src/code/game.c

```diff
@@ -235,28 +239,28 @@ void Game_StateFini(GameState *ctxt) {
     func_801420F4(&D_801F8020);
     func_80141900(&sMonoColors);
     func_80140900(&D_801F8048);
-    THA_Dt(&ctxt->heap);
-    Gamealloc_FreeAll(&ctxt->alloc);
+    THA_Dt(&this->heap);
+    GameAlloc_Cleanup(&this->alloc);
 }
 
-GameStateFunc Game_GetNextStateInit(GameState *ctxt) {
-    return ctxt->nextGameStateInit;
+GameStateFunc GameState_GetInit(GameState* this) {
```

Comment:
With the same logic as not renaming `nextGameStateInit`, I think this should be `GameState_GetNextStateInit`

Reply:
+1